### PR TITLE
First iteration of scraper

### PR DESCRIPTION
**Pull request by rchacon**
Get the ball rolling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+
+## Usage
+
+Copy environment file and update values
+
+```
+cp example.env .env
+```
+
+Run selenium locally rather than in a container. When logging in for the first time you will need to complete a two-step verification process
```
Comment (rchacon) [inline README.md]: newline placement looks weird

File: ally/scraper.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        conn.autocommit = True
+        cursor = conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+
+        for transaction in reversed(transactions):  # Insert older transactions first
+            save_to_db(cursor, transaction)
+
+        cursor.close()
+        conn.close()
+    else:
+        print(json.dumps(transactions, indent=2))
```
Comment (rchacon) [inline ally/scraper.py]: datetime obj is not json serializeable

File: ally/scraper.py [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        ally = AllyScraper(driver, args.username, password, args.account)
+        transactions = ally.run()
+    finally:
+        driver.close()
+
+    if not transactions:
+        sys.exit('No transactions found')
+
+    if args.save:
+        conn = psycopg2.connect(host=os.environ['POSTGRES_HOST'],
```
Comment (rchacon) [inline ally/scraper.py]: consider moving the database stuff to another module

File: ally/scraper.py [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+                if '...%s' % self.account in span.text:
+                    link = WebDriverWait(container, 10).until(
+                        EC.element_to_be_clickable((By.TAG_NAME, 'a'))
+                    )
+                    link.click()
+                    return
+
+        raise Exception('Account %s not found', self.account)
+
+    def get_transactions(self):
```
Comment (rchacon) [inline ally/scraper.py]: Consider only returning transactions that aren't already in database

File: ally/scraper.py (hunk 2 above)
Comment (rchacon1) [reply ally/scraper.py]: fixed

File: ally/scraper.py (hunk 3 above)
Comment (rchacon1) [reply ally/scraper.py]: done

File: ally/scraper.py (hunk 4 above)
Comment (rchacon1) [reply ally/scraper.py]: done

File: README.md (hunk 1 above)
Comment (rchacon) [reply README.md]: fixed

File: ally/scraper.py (hunk 2 above)
Comment (rchacon) [reply ally/scraper.py]: fixed

File: ally/scraper.py (hunk 3 above)
Comment (rchacon) [reply ally/scraper.py]: done

File: ally/scraper.py (hunk 4 above)
Comment (rchacon) [reply ally/scraper.py]: done

**Pull request merged**